manim/mobject/mobject.py

```diff
@@ -96,6 +123,39 @@ def __init__(self, color=WHITE, name=None, dim=3, target=None, z_index=0, **kwar
 
         Container.__init__(self, **kwargs)
 
+    @classmethod
+    def animation_override_for(cls, animation_class):
```

Comment:
Nonblocking but it'd be cool if this could get docs

Reply:
also, typing.